### PR TITLE
[WIP] Baremetal ds bm image

### PR DESCRIPTION
**Pull request by stbenjam**

### CODE REVIEW

File: ci-operator/step-registry/baremetalds/devscripts/gather/baremetalds-devscripts-gather-commands.sh [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+# Fetch packet server IP
+IP=$(cat ${SHARED_DIR}/server-ip)
+export IP
+echo "Packet server IP is ${IP}"
+
+mkdir -p ${ARTIFACT_DIR}
+
+timeout -s 9 15m ssh $SSHOPTS root@$IP bash - << EOF
+oc --insecure-skip-tls-verify adm must-gather --dest-dir ${ARTIFACT_DIR}/must-gather > ${ARTIFACT_DIR}/must-gather/must-gather.log
```
Comment (andfasano) [inline ci-operator/step-registry/baremetalds/devscripts/gather/baremetalds-devscripts-gather-commands.sh]: ${ARTIFACT_DIR} makes sense only in the underlying container executing the step, on the remote Packet server better to use a dedicated folder

File: ci-operator/step-registry/baremetalds/devscripts/gather/baremetalds-devscripts-gather-commands.sh [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+echo "-------[ $SHARED_DIR ]"
+ls -ll ${SHARED_DIR}
+
+# Fetch packet server IP
+IP=$(cat ${SHARED_DIR}/server-ip)
+export IP
+echo "Packet server IP is ${IP}"
+
+mkdir -p ${ARTIFACT_DIR}
```
Comment (andfasano) [inline ci-operator/step-registry/baremetalds/devscripts/gather/baremetalds-devscripts-gather-commands.sh]: This should not be needed, step automatically takes care of it

Comment (stbenjam) [reply ci-operator/step-registry/baremetalds/devscripts/gather/baremetalds-devscripts-gather-commands.sh]: This runs on Packet

Comment (stbenjam) [reply ci-operator/step-registry/baremetalds/devscripts/gather/baremetalds-devscripts-gather-commands.sh]: Well, that was my intention anyway - but yea, it's in the wrong place

File: ci-operator/step-registry/baremetalds/devscripts/gather/baremetalds-devscripts-gather-commands.sh (hunk 1 above)
Comment (stbenjam) [reply ci-operator/step-registry/baremetalds/devscripts/gather/baremetalds-devscripts-gather-commands.sh]: Ah I had just intended to keep it in the same place on the packet server, I meant to do a mkdir -p $ARTIFACT_DIR on packet.  Should we just use a hardcodeded path then?

File: ci-operator/step-registry/baremetalds/e2e/test/baremetalds-e2e-test-commands.sh [hunk 3]
```diff
@@ -85,8 +85,6 @@ echo "### Running tests"
 ssh $SSHOPTS root@$IP openshift-tests run "openshift/conformance/parallel" --dry-run \| grep 'Feature:ProjectAPI' \| openshift-tests run -o /tmp/artifacts/e2e.log --junit-dir /tmp/artifacts/junit -f -
 rv=$?
 
-echo "### Fetching results"
-ssh $SSHOPTS root@$IP tar -czf - /tmp/artifacts | tar -C ${ARTIFACT_DIR} -xzf -
```
Comment (andfasano) [inline ci-operator/step-registry/baremetalds/e2e/test/baremetalds-e2e-test-commands.sh]: This is required to fetch the e2e.log and junit xml produced from the openshift-tests

Comment (stbenjam) [reply ci-operator/step-registry/baremetalds/e2e/test/baremetalds-e2e-test-commands.sh]: This is done in the gather container

Comment (stbenjam) [reply ci-operator/step-registry/baremetalds/e2e/test/baremetalds-e2e-test-commands.sh]: Is there some reason to run scp twice?

**Comment by stevekuznetsov** [comment]
As you're adding OWNERS files in this PR and merging to it constantly, we are spending an inordinate amount of tokens on validating your OWNERS changes. Let's please merge this right now -- even if it's broken -- and iterate on the jobs in a future PR.

**Comment by stbenjam** [comment]
#7623 is the main PR, I'll close this one

/close

**Pull request closed without merging**